thoth_user_api/swagger.yaml

```diff
@@ -606,3 +612,43 @@ definitions:
           type: string
           description: Value of the supplied environment variable.
           example: '1'
+  Metadata:
+    type: object
+    additionalProperties: false
+    properties:
+      apiVersion:
+        type: string
+        description: >
+          BuildLog api version.
+        example: >
+          "buildlog.thoth-station.ninja/v1"
+      kind:
+        type: string
+        description: >
+          Type of log.
+        example: >
+          kind - buildlog
```

Comment:
I think this should be "BuildLog" only, not "kind - buildlog"

Reply:
ja, the value is BuildLog, il change this.